tests/jballerina-unit-test/src/test/resources/test-src/types/xml/xmlns-def.bal

```diff
@@ -0,0 +1,26 @@
+type Rec record {|
+    string a = ns1:baz;
+|};
+
+xmlns "http://ballerina.com/b" as ns1;
+
+function testXMLNSDefinition() {
+    string s1 = ns0:foo;
+    assert(s1, "{http://ballerina.com/a}foo");
+
+    string s2 = ns1:bar;
+    assert(s2, "{http://ballerina.com/b}bar");
+
+    Rec rec = {};
+    assert(rec.a, "{http://ballerina.com/b}baz");
+}
+
+xmlns "http://ballerina.com/a" as ns0;
```

Comment:
Shall we add few test where, `xmlns [CONST] as ns0;` and

Reply:
Using const for the namespaceURI does not work at the moment. There is a PR ([41958](https://github.com/ballerina-platform/ballerina-lang/pull/41958)) to address this issue. Once that gets merged we can add a test.